libraries/AP_IOMCU/iofirmware/analog.cpp

```diff
@@ -157,3 +159,54 @@ OSAL_IRQ_HANDLER(STM32_ADC1_HANDLER) {
 
     OSAL_IRQ_EPILOGUE();
 }
+
+#else
+#include <AP_HAL/AP_HAL.h>
+
+extern const AP_HAL::HAL& hal;
+
+AP_HAL::AnalogSource *vrssi;
+
+void adc_init(void)
+{
+    vrssi = hal.analogin->channel(HAL_IOMCU_RSSI_ADC_CHANNEL);
+}
+
+/*
+  capture VSERVO in mV
```

Comment:
Comments are wrong. Should add a comment as to why these are empty

Reply:
fixed